app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,33 @@
+class CommentsController < ApplicationController
+  before_action :find_user
+  before_action :find_post
+
+  def new
+    @comment = @post.comments.new
+  end
+
+  def create
+    @comment = @post.comments.new(comment_params)
+    @comment.author = @user
+    if @comment.save
+      flash[:notice] = 'Comment created successfully.'
+      redirect_to user_post_path(@user, @post)
+    else
+      render 'new'
+    end
+  end
```

Comment:
- [ ] When i try creating a new comment, the below error pops up. The reason is you have not created a form in views/comments/new for creating a new comment. Remember rails follow the MVC pattern so is advisable to follow this pattern when you are coding your project in order to avoid errors. ⬇️ 
![image](https://github.com/Wils-FOURR-speed/Blog-App-Project/assets/32361963/11b4e469-8bac-4a27-a81a-cde90d4b0ec1)